Dockerfile

```diff
@@ -22,7 +22,8 @@ FROM base as deps
 # Copy and install Unstructured
 COPY requirements requirements
 
-RUN python3.10 -m pip install pip==${PIP_VERSION} && \
+RUN export ARCH=$(uname -m) && \
```

Comment:
should we move the docker file changes to base image since unstructured and unstructured  api images both need it?

Reply:
Unfortunately no, cause it needs to be installed during building of our unstructured images as the base images are built on arm64 so our wheel would not get installed.  It's a little duplication between unstructured and unstructured-api but it should stay minimal